src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/model/TaskProgress.kt

```diff
@@ -9,6 +9,6 @@ data class TaskProgress(
   @Schema(description = "The name of an outstanding task", example = "ASSESS_ELIGIBILITY")
   val name: Task,
 
-  @Schema(description = "The state of this task for a specific assessment", example = "Smith")
+  @Schema(description = "The state of this task for a specific assessment", example = "Amexar")
```

Comment:
mmh.. this doesn't look right.. it wasn't before either

Reply:
Yup just did another browse through the files and saw this. I will change.